frontend/src/app/workflows/download-es/download-es.component.html

```diff
@@ -0,0 +1,45 @@
+<app-modal #modal="modal" [back]="['..']" [backOptions]="{relativeTo: route}">
+  <ng-container modal-header>
+    Configure and Download
+  </ng-container>
+  <ng-container modal-body>
+    <div class="col">
+      <div class="mb-3 form-check">
+        <input type="checkbox" class="form-check-input" id="yamlCheck" [(ngModel)]="exportOptions.yaml">
+        <label class="form-check-label" for="yamlCheck">YAML (Codemirror Content)</label>
```

Comment:
Users don't know what Codemirror is. Should be `Editor content`

Reply:
Agreed